Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build Hapistrano
+FROM fpco/stack-build as build-env
```

Comment:
@javcasas what about using `haskell:8.2` instead?

| Image | Size |
| --- | --- |
| fpco/stack-build:latest | 3 GB |
| haskell:8.2 | 266 MB |

Reply:
Thet Haskell image uses rather old libraries (specially old gcc), and will need a lot of dependencies, but if it works then better so.

Reply:
I'll try that one.